pkgs/development/ocaml-modules/pfff/default.nix

```
@@ -0,0 +1,90 @@
+{ buildDunePackage
+, lib
+, fetchFromGitHub
+, ocaml
+, opam
+, perl
+, dune_2
+, ocamlgraph
+, yojson
+, menhir
+, grain_dypgen
+, ppx_deriving
+, ppx_hash
+, ppxlib
+, uutf
+, uucp
+, alcotest
+, ansiterminal
+, easy_logging
+, easy_logging_yojson
+}:
+
+buildDunePackage rec {
+  pname = "pfff";
+  version = "2021-12-14";
+
+  src = fetchFromGitHub {
+    owner = "returntocorp";
+    repo = "pfff";
+    rev = "0a3be7857551b69c430dc081575f9daef8ac85a0";
+    sha256 = "YA/BN6AFXLyeT3SxCixqtzKgd4LfrDNqw1IEEsHDs+c=";
+  };
+
+  nativeBuildInputs = [
+    ocaml
+    opam
+    perl
+    dune_2
+    menhir
+    grain_dypgen
+  ];
+
```

```suggestion
  nativeBuildInputs = [
    opam
    perl
    menhir
    grain_dypgen
  ];
```

Already added by `buildDunePackage`